src/Compilers/CSharp/Test/Semantic/Semantics/BindingAsyncTests.cs

```diff
@@ -3857,5 +3857,28 @@ static void M()
             var compilation = CreateCompilationWithMscorlib45(source, references: new[] { reference });
             compilation.VerifyEmitDiagnostics();
         }
+
+
+        [Fact]
+        [WorkItem(13734, "https://github.com/dotnet/roslyn/issues/13734")]
+        public void TypeofAwaitedTask()
+        {
+            string source =
+    @"
+using System.Threading.Tasks;
+using System;
+
+namespace AsyncBug
+{
+    public class Foo {
+        public async void Boom() {
+            Func<Type> f = (await Task.FromResult(1)).GetType;
```

Comment:
Now that I think about it, this test case should probably be grouped with the other async emit tests.